applications/crossbar/doc/ref/limits.md

```diff
@@ -14,6 +14,7 @@ Key | Description | Type | Default | Required | Support Level
 `burst_trunks` | The number of two-way, flat-rate trunks used only if no other trunks are available | `integer()` |   | `false` | `beta`
 `calls` | A hard limit for the total number calls | `integer()` |   | `false` | `beta`
 `inbound_trunks` | The number of inbound, flat-rate trunks | `integer()` |   | `false` | `supported`
+`onnet_exempt` | Determines if onnet-terminated calls should be exempt from billing (kazoo account to kazoo account calls) | `boolean()` | `false` | `false` | `beta`
```

Comment:
There are two versions of the "docs"; here in the `doc/ref` folder KAZOO auto-generates a "reference" doc with what is auto-detected from schemas and source code changes.

Up a directory, you should find `doc/limits.md`; that is the real doc that the docs site displays when built. So you should migrate any changes here to that doc to keep them in sync, as well as any additional explanatory text needed to make sense of what's changing in the code.